security.rst

```diff
@@ -10,28 +10,22 @@ Security
     Do you prefer video tutorials? Check out the `Symfony Security screencast series`_.
 
 Symfony's security system is incredibly powerful, but it can also be confusing
-to set up. In this article you'll learn how to set up your application's security
-step-by-step, from configuring your firewall and how you load users, to denying
-access and fetching the User object. Depending on what you need, sometimes
-the initial setup can be tough. But once it's done, Symfony's security system
-is both flexible and (hopefully) fun to work with.
+to set up. Don't worry! In this article, you'll learn how to set up your app's
+security system step-by-step:
 
-Since there's a lot to talk about, this article is organized into a few big
-sections:
+#. :ref:`Installing security support <security-installation>`;
 
-#. Installing security support;
+#. :ref:`Create your User Class <create-user-class>`;
 
-#. Initial ``security.yaml`` setup (*authentication*);
+#. :ref:`*Authentication* & Firewalls <security-yaml-firewalls>`;
```

Comment:
these stars, here and below, stay in the output, while the previous logic was using them to make it bold. Is it expected ?

Reply:
Thanks for reporting! It's being fixed in #10513.